lib/pentagon/components/vpc/files/Makefile

```diff
@@ -1,19 +0,0 @@
-.PHONY: all plan apply destroy provision ssh
-
-all: plan apply provision
-
-plan:
-	terraform init -backend=true
-	terraform plan -module-depth=-1 -var-file terraform.tfvars
-
-apply:
-	terraform apply
-
-destroy:
-	terraform plan -destroy -var-file terraform.tfvars -out terraform.tfplan
-	terraform apply terraform.tfplan
-
-clean:
-	rm -f terraform.tfplan
-	rm -f terraform.tfstate
-	rm -fR .terraform/
```

Comment:
I'm going to remove it, (but open to suggestions). My thinking is that since templat-ing for the 'vpc' it's built-in to pentagon via `lib/__init__.py`, versus being installed as an explicit component.